includes/payment-methods/class-wc-stripe-ocs-payment-gateway.php

```
@@ -0,0 +1,404 @@
+	 *
+	 * Distinct from {@see self::is_valid_optimized_checkout_page()} — which gates *rendering*
+	 * to actual checkout pages. This broader check is true wherever OCS-aware behavior
+	 * should apply (for example, My Account → Payment Methods, where saved tokens for
+	 * sub-gateways must still surface under the consolidated `stripe` gateway).
+	 *
+	 * @return bool
+	 */
+	public function is_optimized_checkout_active(): bool {
+		return ! $this->is_on_add_payment_method_page()
+			&& ! $this->is_changing_payment_method_for_subscription();
+	}
```

PHPMD (2.15.0)

[error] The method is_optimized_checkout_active is not named in camelCase. (undefined)

(CamelCaseMethodName)